sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
   - `offset` is the offset of the text from the start of the document
   
```

Comment:
TextAnalyticsErrorException needs to be deleted?

Reply:
Good catch. Yes, `TextAnalyticsErrorException` in the implementation folder can be deleted since swagger no longer have it.